src/StoreInjector.ts

```diff
@@ -41,60 +40,40 @@ export function storeInject<S>(config: StoreInjectConfig<S>) {
 
 	return handleDecorator((target, propertyKey) => {
 		beforeProperties(function(this: WidgetBase & { own: Function }, properties: any) {
-			const injector: StoreInjector | null = this.registry.getInjector(name);
-			if (injector) {
+			const injectorItem = this.registry.getInjector<Store<S>>(name);
+			if (injectorItem) {
+				const { injector } = injectorItem;
+				const storeInjector = injector();
```

Comment:
Is this still a storeInjector or just a store at this point?

Reply:
done!